<launch>

  <param name="action_server_url" type="str" value="http://localhost:5555/webhook"/>
  <param name="action_server_port" type="int" value="5555"/>
  <param name="dialogue_res_path" type="str" value="$(find dialogue_system)/res"/>

  <node name="dialogue_system" pkg="dialogue_system" type="dialogue_system.py" respawn="false" output="screen"/>
</launch>
